### PR TITLE
Fix loading text if naturalEvents is not set

### LINKED ISSUE #627: "Loading..." is not removed from sidebar if naturalEvents feature is disabled
The loading text in the sidebar is not removed if naturalEvents is disabled https://github.com/nasa-gibs/worldview/blob/a85692d5f20cc227dfefaf0cca7a5e905c0b3de0/web/index.html#L294

The code seems inconsistent as wv-events is dedicated to the Events tab but it's also used for Loading information. So if naturalEvent is disabled the wv-events div is not filled and Loading text is still displayed after rendering.
https://github.com/nasa-gibs/worldview/blob/a85692d5f20cc227dfefaf0cca7a5e905c0b3de0/web/js/layers/wv.layers.sidebar.js#L187-L198

### PR DESCRIPTION
**Pull request by PacoDu**
Fixes #627 .

Changes in this pull request:

- Split div wv-events into wv-events and wv-loading for consistency
- Added loading hide() after sidebar rendering 

@nasa-gibs/worldview

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -291,7 +291,8 @@
               <ul id="productsHolder-tabs"></ul>
               <div id="productsHoldertoggleButtonHolder"></div>
               <div id="products"></div>
-              <div id="wv-events">Loading...</div>
+              <div id="wv-events"></div>
+              <div id="wv-loading">Loading...</div>
```
[the remark this answers is not in the document]
Comment (PacoDu) [reply web/index.html]: Yes I agree, I'll push that in a minute

**Comment by localjo** [comment]
I originally added the "Loading..." text to fix an issue; when switching to the events tab, there is a delay while events information loads and the tab was empty with no indication that anything was loading. On some connections, it would take several seconds for anything to display in the tab, which made it seem like there were no events and would lead to users switching tabs again before anything loaded. You can reproduce that issue by setting your dev tools to emulate a slow connection, disabling cache, reloading the page and then clicking on the events tab. On `master` you should see "Loading..." until the events load. This PR removes that text from the events tab, so it reintroduces the original problem.

**Comment by PacoDu** [comment]
Oh ok I see, so should I just add an else condition after https://github.com/nasa-gibs/worldview/blob/a85692d5f20cc227dfefaf0cca7a5e905c0b3de0/web/js/layers/wv.layers.sidebar.js#L199-L210

Something like: 
```
else{
   $('#wv-events').hide()
}
```

**Comment by Benjaki2** [comment]
@PacoDu I think that condition could go here for a quick fix: https://github.com/nasa-gibs/worldview/blob/master/web/js/wv.main.js#L147-L149

But it would probably be better to take the `Loading...` text out of the `index.html` and actually append that when [wv.naturalEvents.ui](https://github.com/nasa-gibs/worldview/blob/master/web/js/naturalEvents/wv.naturalEvents.ui.js) is initiated.

**Comment by PacoDu** [comment]
@Benjaki2 Yes the code would be more consistent. I don't have time now but I'll take a look tomorrow.

**Comment by PacoDu** [comment]
Well, sorry for the delay :). 
I've moved the Loading string to naturalEvents init, this Fixes #627 and still display "Loading..." on NaturalEvent tab for slow connections (tested with 3G network settings on chrome).

**Comment by Benjaki2** [comment]
@PacoDu We are in the process of merging and releasing #503 which should be on Production Monday (if all goes as planned :crossed_fingers: ). We will try to get this merged in right after. Thanks for the contribution!

File: web/js/naturalEvents/wv.naturalEvents.ui.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ wv.naturalEvents.ui = wv.naturalEvents.ui || function (models, ui, config, reque
   var naturalEventMarkers = wv.naturalEvents.markers(models, ui, config);
 
   var init = function () {
+    // Display loading information for user feedback on slow network
+    $('#wv-events').text('Loading...');
```
Comment (localjo) [inline web/js/naturalEvents/wv.naturalEvents.ui.js]: This file has changed in the `module-loaders` branch. Let's try to change the target of this pull request to that branch, and manually adjust it if necessary.

**Pull request merged**